packages/x-date-pickers-pro/src/DateRangePicker/DateRangePickerViewDesktop.tsx

```diff
@@ -182,16 +187,16 @@ export function DateRangePickerViewDesktop<TDate>(props: DesktopDateRangeCalenda
         return (
           <DateRangePickerViewDesktopContainer key={index}>
             <DateRangePickerViewDesktopArrowSwitcher
-              onLeftClick={selectPreviousMonth}
-              onRightClick={selectNextMonth}
-              isLeftHidden={index !== 0}
-              isRightHidden={index !== calendars - 1}
-              isLeftDisabled={isPreviousMonthDisabled}
-              isRightDisabled={isNextMonthDisabled}
-              leftArrowButtonText={leftArrowButtonText}
+              goToPrevious={selectPreviousMonth}
```

Comment:
Nitpick: Don't we usually follow convention to prefix `actions` with `on` to make it more clear, what kind of prop is this?
I.e.: `onPreviousClick`, `onNextClick`

Reply:
I liked the idea of focusing on the behavior (go to the previous / next item) rather than focusing on the user action (click on a button).
Even if, for this component, both are probably 100% equivalent.

Reply:
I do understand and am pro for this change.
I was simply asking about the prop naming convention in regards to `on` prefix for actions, like `onChange`, `onClick`, `onGoToNext` or like I said `onNextClick`

Reply:
For the convention, maybe `onGoToNext`.

I just want to replace the `click` part by a `go to` part.

Reply:
Done :+1: